packages/ocs/dashboards/persistent-internal/capacity-trend-card/capacity-trend-card.tsx

```diff
@@ -51,20 +50,26 @@ const calculateDaysUp = (response: PrometheusResponse): number | null => {
     return null;
   }
 
-  const values: PrometheusValue[] = response.data?.result?.[0]?.values || [];
+  let minTimestamp: number;
+  let maxTimestamp: number = 0;
 
-  if (!values.length) {
-    return null;
-  }
+  for (const promResult of response.data.result) {
+    if (!promResult.values.length) {
+      // eslint-disable-next-line no-continue
+      continue;
+    }
 
-  const timestamps: number[] = values.map(([timestamp, _value]) => timestamp);
+    const timestamps: number[] = promResult.values.map(
+      ([timestamp, _value]) => timestamp
+    );
 
-  const minTimestamp: number = Math.min(...timestamps);
-  const maxTimestamp: number = Math.max(...timestamps);
+    minTimestamp = Math.min(...timestamps, minTimestamp ?? timestamps[0]);
```

Comment:
nits: why not just default `minTimestamp` to `0` like `maxTimestamp` ?
```suggestion
    minTimestamp = Math.min(...timestamps, minTimestamp);
```

Reply:
0 is the least value, if se set min to that. it'll always be least

Reply:
lol, my bad...

Reply:
https://github.com/red-hat-storage/odf-console/pull/1312/files#r1571929991